packages/antd-audit-log/package.json

```
@@ -0,0 +1,60 @@
+{
+    "name": "@pankod/refine-antd-audit-log",
+    "description": "refine audit log component for antd package. refine is a React-based framework for building internal tools, rapidly. It ships with Ant Design System, an enterprise-level UI toolkit.",
+    "version": "3.8.0",
+    "license": "MIT",
+    "main": "dist/index.js",
+    "typings": "dist/index.d.ts",
+    "private": false,
```

```suggestion
    "private": true,
```
we can mark as a private for disable npm publish